### ISSUE TITLE
Falsche Werte /Wertedefinition in den HAM - Datenpunkten

### ISSUE THREAD

**Issue by BramTown** [ioBroker/ioBroker.ham#140]
Seit Umstellung auf den js-contoller v3.3x werden falsche Werte bzw. falsche Definitionen von Datenpunkten des HAM-Adapters im Log als Warnung angegeben
```

ham.0 | 2021-05-01 13:52:12.458 | warn | (14212) State value to set for "ham.0.DAZN.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
-- | -- | -- | --
ham.0 | 2021-05-01 13:52:12.457 | warn | (14212) State value to set for "ham.0.Eurosport2.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.455 | warn | (14212) State value to set for "ham.0.Eurosport1.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.454 | warn | (14212) State value to set for "ham.0.Sky-Action.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.452 | warn | (14212) State value to set for "ham.0.Sky-Cinema.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.451 | warn | (14212) State value to set for "ham.0.Sky-Bundesliga-UHD.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.449 | warn | (14212) State value to set for "ham.0.Sky-Bundesliga-HD-1.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.447 | warn | (14212) State value to set for "ham.0.Sky-Sport-HD-2.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.446 | warn | (14212) State value to set for "ham.0.Sky-Sport-HD-1.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.444 | warn | (14212) State value to set for "ham.0.Sky-Sport-UHD.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.435 | warn | (14212) State value to set for "ham.0.Das-Dritte.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.434 | warn | (14212) State value to set for "ham.0.Das-Zweite.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.432 | warn | (14212) State value to set for "ham.0.Das-Erste.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.431 | warn | (14212) State value to set for "ham.0.SkyRecord.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.429 | warn | (14212) State value to set for "ham.0.SkyStop.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.427 | warn | (14212) State value to set for "ham.0.SkyPause.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.421 | warn | (14212) State value to set for "ham.0.SkyPlay.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.419 | warn | (14212) State value to set for "ham.0.SkyHome.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.418 | warn | (14212) State value to set for "ham.0.SkyCurserRight.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.416 | warn | (14212) State value to set for "ham.0.SkyCurserLeft.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.415 | warn | (14212) State value to set for "ham.0.SkyCurserDown.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"
ham.0 | 2021-05-01 13:52:12.413 | warn | (14212) State value to set for "ham.0.SkyCurserUp.Accessory-Information.Identify" has wrong type "undefined" but has to be "boolean"


```

**Comment by UncleSamSwiss** [comment]
My assumption would be that we need to replace `undefined` with `null` as this should still be allowed.

**Comment by Apollon77** [comment]
or simply not set them in this case because they basically have no value (and for all Eccessory information will also never really get one if not there initially)

**Comment by Apollon77** [comment]
@BramTown Please test GitHub version. Version number stay same, so restart manually after the install. Errors gone?

**Comment by Apollon77** [comment]
fix will come as 4.0.2
